vsintegration/src/FSharp.Editor/Classification/ClassificationService.fs

```diff
@@ -20,7 +22,42 @@ open Microsoft.CodeAnalysis.ExternalAccess.FSharp.Classification
 // IVT, we'll maintain the status quo.
 #nowarn "44"
 
+#nowarn "57"
+
+open FSharp.Compiler.Range
 open FSharp.Compiler.SourceCodeServices
+open FSharp.Compiler.SourceCodeServices.Lexer
+
+type SemanticClassificationData = (struct(FSharp.Compiler.Range.range * SemanticClassificationType)[])
```

Comment:
You can drop the `FSharp.Compiler.Range.`

Reply:
Sure